_specification-v1/components.md

```diff
@@ -80,87 +75,45 @@ A component's folder structure **may** be organised as follows. The following is
 
-### Bower
+### npm
 
-Origami components **must** be installable through the <a href="https://bower.io/" class="o-typography-link--external">Bower package manager</a>, and **should** include a `bower.json` manifest file which configures the component. The component **should not** be published to the public Bower registry (through `bower register` or similar).
+Origami components **must** be installable through the <a href="https://npmjs.com/" class="o-typography-link--external">npm package manager</a>, and **must** include a `package.json` manifest file which configures the component. As well as following the <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json" class="o-typography-link--external">`package.json` spec</a>, there are additional requirements to make the component's npm manifest conform to the Origami specification:
+
+  - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#name">`name`</a> property set to the package name, e.g. `@financial-times/o-typography`.
+  - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#browser">`browser`</a> property set to the component's main JavaScript file (`main.js`) **_if_** it exists.
+  - It **must** include a <a href="https://nodejs.org/api/packages.html#packages_type">`type`</a> property set to `"module"`.
+  - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#license">`license`</a> property set to the <a href="https://spdx.org/licenses/" class="o-typography-link--external">SPDX license identifier</a> for the license the Origami component is using, e.g. `"MIT"`.
+  - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#devdependencies">`devDependencies`</a> property set to an object **_if_** the component has any npm dependencies required for development or testing.
+
+  - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependencies">`peerDependencies`</a> property set to an object **_if_** the component has any production sass dependencies or production dependencies which are also Origami components.
+  - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies">`dependencies`</a> property set to an object **_if_** the component has any production javascript dependencies which are not also Origami components.
+  - It **must** include an <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines">`engines`</a> property set to an object which has an `npm` property set to a SemVer range which disallows anything below `v7.0.0`. E.G. `^7` would be valid as would `>= 7`.
+  - It **may** contain a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#description">`description`</a> property set to a short description of the component.
```

Comment:
Wondering if we should mention `version`. It must be set according to the npm spec but when authoring components we'll be setting `0.0.0` as the version is updated in CI when the bundle is published. Shall we not mention it in the spec but talk about it in a tutorial?

Reply:
> Shall we not mention it in the spec but talk about it in a tutorial?

Yes, this sounds good to me 👍